### PR TITLE
Add Neutralizing Gas message, display Manaphy egg gacha origin, allow catching wonder guard bosses

### PR DESCRIPTION
**Pull request by TempsRay**
* Neutralizing Gas should display a message upon being sent out so I added a `PostSummonMessageAbAttr` to it
* Manaphy eggs currently are hardcoded to not display where they were obtained. This is purely a cosmetic bug since the game still keeps track of their `gachaType` (an enum) while not reflecting a string description of it for the egg-list-ui-handler, confusing players. [Example of what it looks like](https://cdn.discordapp.com/attachments/1233819365510615050/1233891723495870494/image.png?ex=662ebeb7&is=662d6d37&hm=8d544d139c97acf7222c5edd41970d93cc427c683f672fad14a31914166706c0&)
* There is an edge case where a boss Pokemon has wonder guard in which case it is uncatchable without a master ball

### CODE REVIEW

**Comment by Flashfyre** [comment]
Resolve conflicts

**Comment by TempsRay** [comment]
conflicts resolved

File: src/phases.ts [hunk 1]
```diff
@@ -1722,7 +1723,7 @@ export class CommandPhase extends FieldPhase {
             }, null, true);
           } else if (cursor < 5) {
             const targetPokemon = this.scene.getEnemyField().find(p => p.isActive(true));
-            if (targetPokemon.isBoss() && targetPokemon.bossSegmentIndex >= 1 && cursor < PokeballType.MASTER_BALL) {
+            if (targetPokemon.isBoss() && targetPokemon.bossSegmentIndex >= 1 && !targetPokemon.hasAbility(Abilities.WONDER_GUARD) && cursor < PokeballType.MASTER_BALL) {
```
Comment (Xavion3) [inline src/phases.ts:1726]: Should probably be `.hasAbility(Abilities.WONDER_GUARD, false, true)` so it doesn't care if wonder guard is currently suppressed or overridden.

**Review by Xavion3 — CHANGES_REQUESTED** [review]

File: src/phases.ts (hunk 1 above)
Comment (TempsRay) [reply src/phases.ts:1726]: Gotcha, will change

**Review by Xavion3 — APPROVED** [review]

**Pull request merged**